app/src/main/java/com/example/movietrackerapp/model/remote/datasource/MovieServiceRemoteDataSourceImpl.kt

```diff
@@ -1,5 +1,27 @@
 package com.example.movietrackerapp.model.remote.datasource
 
-class MovieServiceRemoteDataSourceImpl: MovieServiceRemoteDataSource {
+import android.content.Context
+import com.example.movietrackerapp.R
+import com.example.movietrackerapp.model.remote.service.MovieService
+import com.example.movietrackerapp.model.remote.service.response.MovieResponse
+import com.example.movietrackerapp.utils.ResponseWrapper
 
+class MovieServiceRemoteDataSourceImpl(
+    private val movieService: MovieService,
+    private val context: Context
+): MovieServiceRemoteDataSource {
+    override suspend fun getMovieRecommendations(): ResponseWrapper<MovieResponse> {
+        return try {
+            val response = movieService.getRecommendations(context.getString(R.string.api_key))
```

Comment:
Why you set the key in an xml file ? I mean is it more secured ? in my case I would just declared a const val variable under an object class.